### PR TITLE
caching doc review

### PR DESCRIPTION
**Pull request by woodp**

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/caching.md](https://github.com/woodp/architecture-center/blob/caching-review/docs/best-practices/caching.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/caching?branch=pr-en-us-63) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C26%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704262052148976-63%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/caching.md](https://github.com/woodp/architecture-center/blob/caching-review/docs/best-practices/caching.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/caching?branch=pr-en-us-63) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C27%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201704271654556719-63%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/best-practices/caching.md [hunk 1]
```diff
@@ -467,7 +467,7 @@ in the cloud.
 > are currently slated for retirement on November 16th, 2016.
 > It is recommended that you migrate to Azure Redis Cache in
 > preparation for this retirement. For more information, see
-> [What is Azure Redis Cache offering and what size should I use?](/azure/redis-cache/cache-faq#what-redis-cache-offering-and-size-should-i-use)e.
+> [What is Azure Redis Cache offering and what size should I use?](https://docs.microsoft.com/en-us/azure/redis-cache/cache-faq#what-redis-cache-offering-and-size-should-i-use).
```
Comment (MikeWasson) [inline docs/best-practices/caching.md]: Links to docs.microsoft.com should just use the segment (/azure/redis-cache/....)

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/best-practices/caching.md](https://github.com/woodp/architecture-center/blob/caching-review/docs/best-practices/caching.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/best-practices/caching?branch=pr-en-us-63) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C24%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705241752364944-63%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**